src/freenet/client/filter/ElementInfo.java

```diff
@@ -218,7 +227,27 @@ public class ElementInfo {
 			"first-letter",
 			"before",
 			"after",
-			"target"
+			"target",
+			"any-link",
+			"default",
+			"defined",
```

Comment:
we do not have custom element support (it requires javascript), so defined does not make sense. Please remove it.

Reply:
Remove it or add it into the BANNED_PSEUDOCLASS?

Reply:
BANNED_PSEUDOCLASS is better: CSSTokenizerFilter::HTMLElementVerifier defines that as `banned (but otherwise valid) selector`.